compiler/src/test/scala/org/scalajs/core/compiler/test/JSInteropTest.scala

```diff
@@ -303,25 +345,12 @@ class JSInteropTest extends DirectTest with TestHelpers {
     } {
       val expectedMessage = {
         s"""
-          |newSource1.scala:7: error: Native JS classes and objects can only have one annotation among JSName, JSGlobal, JSImport and JSGlobalScope.
+          |newSource1.scala:7: error: Native JS classes and objects can only have one annotation among JSGlobal, JSImport and JSGlobalScope.
           |$secondAnnot
           | ^
         """
```

Comment:
Consider inlining the rhs of `expectedMessage` where it is used, as an argument to `hasErrors`, now that it doesn't need to be composed with `jsNameWarning` anymore.